README.md

```diff
@@ -720,9 +720,9 @@ Translations of the guide are available in the following languages:
 ## Syntax
 
 * <a name="double-colons"></a>
-    Use `::` only to reference constants(this includes classes and
+    Use `::` only to reference constants (this includes classes and
     modules) and constructors (like `Array()` or `Nokogiri::HTML()`).
-    Do not use `::` for regular method invocation.
+     * Do not use `::` for regular method invocation.
```

Comment:
Why did you convert this into a bullet point?

Reply:
Sorry, I should have explained my thought process. The first sentence sounds all encompassing about usages of `::`.
> Use `::` only to reference constants (this includes classes and
 modules) and constructors (like `Array()` or `Nokogiri::HTML()`).

It felt like ```Do not use `::` for regular method invocation``` and ```Do not use `::` to define class methods``` were supporting examples of where not to use double colons. I can see where this may not make sense, and I am open to suggestions on how to change this.